cabal-install/cabal-install.cabal

```diff
@@ -189,6 +189,11 @@ Flag network-uri
   description:  Get Network.URI from the network-uri package
   default:      True
 
+Flag debug-assertions
```

Comment:
Bikeshedding: what about introducing a flag `assertions` and call this one `expensive-assertions` to make it more clear what's going on? Then I guess `debugAssert` could be renamed to `expensiveAssert`. What do you think?